packages/breadcrumbs/src/styled/StyledChevronIcon.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useContext, SVGAttributes } from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import ChevronLeftStrokeIcon from '@zendeskgarden/svg-icons/src/12/chevron-left-stroke.svg';
+import ChevronRightStrokeIcon from '@zendeskgarden/svg-icons/src/12/chevron-right-stroke.svg';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const LTR_COMPONENT_ID = 'breadcrumbs.chevron_right_stroke_icon';
+const RTL_COMPONENT_ID = 'breadcrumbs.chevron_left_stroke_icon';
+
+const getGardenId = (rtl: boolean) => (rtl ? RTL_COMPONENT_ID : LTR_COMPONENT_ID);
+
+const ChevronStrokeIcon = (props: SVGAttributes<SVGElement>) => {
+  const { rtl } = useContext(ThemeContext);
+
+  return rtl ? <ChevronLeftStrokeIcon {...props} /> : <ChevronRightStrokeIcon {...props} />;
+};
+
+/**
+ * Accepts all `<svg>` props
+ */
+export const StyledChevronIcon = styled(ChevronStrokeIcon).attrs(props => ({
+  'data-garden-id': `${getGardenId(props.theme.rtl)}`,
+  'data-garden-version': PACKAGE_VERSION,
+  'aria-hidden': 'true'
+}))`
+  display: inline-block;
+  position: relative;
+  top: 1px;
+  margin: 0 calc(8 / 14 * 1em);
```

Comment:
Looks like you've got yourself backed into a corner with this one. Whenever you find yourself "pixel-nudging" it's a good indication that it's time to step back and look at the problem differently. The breadcrumb is already flexbox, so it seems like this should just be a flex item with appropriate `align-self` applied.

Reply:
Thanks for bringing this up, I will be more aware of avoiding pixel pushing in the future. The `<ol>` is flexbox but applying various `align-self` values did not center the chevron. However, I applied `display:flex;` and `align-items:center;` to the list items and that got the chevrons centered.